users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -47,15 +47,20 @@ def get_datasets(**kwargs):
         returnn_python_exe=RETURNN_EXE,
         **kwargs,
     )
-
```

Comment:
I think the empty lines give a bit more structure to the function's content, so I'd keep them.

Reply:
even if black disagrees?